File: app/src/main/java/com/neome/java/api/meta/base/dto/EntVdApplyTransforms.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdVdAutoFunc;

import java.util.Map;

@SuppressWarnings("unused")
public class EntVdApplyTransforms extends EntVdAutoStep
{
  public MetaIdVdAutoFunc[] keys;

  public Map<MetaIdVdAutoFunc, AutoXform> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdApplyTransforms.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.AutoXform
import com.neome.api.meta.base.dto.EntVdAutoStep
import com.neome.api.meta.base.Types.MetaIdVdAutoFunc

interface EntVdApplyTransforms : EntVdAutoStep
{
  val keys: Array<MetaIdVdAutoFunc>
  val map: Map<MetaIdVdAutoFunc, AutoXform>
}